contrib/bloop/src/mill/contrib/bloop/BloopImpl.scala

```diff
@@ -106,14 +102,29 @@ class BloopImpl(ev: () => Evaluator, wd: Path) extends ExternalModule { outer =>
     }
   }
 
+  // Compute all transitive modules from build children and via moduleDeps
+  def transitiveModules(mod: define.Module, found: Seq[define.Module] = Seq.empty): Seq[define.Module] = {
```

Comment:
Note: The explicit `found` parameter looks a bit like the intention was to make this `def` tail recursive, but it isn't (yet).

Reply:
Actually I wasn't aiming for tail recursion. I would hope no one would have enough modules to require it. I just wanted to avoid infinite recursion when two modules are declared co-dependent.